src/Avalonia.FuncUI/DSL/Base/StyledElement.fs

```diff
@@ -5,11 +5,165 @@ open Avalonia.Controls.Primitives
 [<AutoOpen>]
 module StyledElement =  
     open Avalonia
+    open Avalonia.FuncUI
     open Avalonia.FuncUI.Types
     open Avalonia.FuncUI.Builder
     open Avalonia.Styling
+    open Avalonia.LogicalTree
+    open System.Threading
+    open System
+
+    module internal ClassesInternals =
+        open System.Linq
+
+        /// pseudoclasse is classe beginning with a ':' character.
+        let isPseudoClass (s: string) = s.StartsWith(":")
```

Comment:
s.StartsWith(":") -> s.StartsWith(':') ?

Reply:
Exactly, I've made the correction.